tests/configurable_cluster.py

```diff
@@ -140,6 +149,11 @@ def get_down_hostname(self, host_name):
 
     def exec_cmd_on_host(self, host, cmd, user=None, raise_error=True,
                          tty=False):
+        # Ensure command is invoked with sudo since EMR's login user is ec2-user
+        # If sudo is already present in the command then no error will occur
+        # as arbitrary nesting of sudo is allowed
```

Comment:
Also, base_product_case.run_prestoadmin uses this. This will mean that we're always running presto-admin as sudo, which will mask any errors we want to ferret out in our transition to not requring root.